features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdKafkaDecoder.java

```diff
@@ -0,0 +1,279 @@
+package org.opennms.netmgt.trapd;
```

Comment:
HZN-766: Replace this code with JAXB (de)serialization of the TrapNotification object.